lib/matplotlib/contour.py

```diff
@@ -1088,7 +1088,7 @@ def _autolev(self, N):
         """
         if self.locator is None:
             if self.logscale:
-                self.locator = ticker.LogLocator()
+                self.locator = ticker.LogLocator(numticks=N)
```

Comment:
Should this be `N+1` like the line below?

The docstring of this function will also need to be changed (as it claims that this is not supporte).

Reply:
I think, purpose of N is to set the level to `N` , idk if it should be `N+1` I would like to ask @jklymak  to weigh in! Regarding docstring, I guess it should have been doing this thing based on `N` but yeah I can update the docstring, if you all would suggest that!

Reply:
Sorry, I don't have any particular input here about the "right" behaviour.  I think you should just make it consistent with what LogLocator would do an an axes.  You will either break some tests with this, or if there were no tests will need to create some tests that exercise this...

In the above the changes are quite large, so you need to argue that they are better, or more flexible somehow.  Maybe the default needs to be something that is as backwards compatible as possible.  

Overall, though, I think passing numticks through to the LogLocator makes sense, but you should argue it a bit more completely here to get buy-in.